src/main/pages/che-6/portable-workspaces/factories_json_reference.adoc

```diff
@@ -91,87 +88,106 @@ factory.workspace.project : {
 [id="policies"]
 == Policies
 
+Following is an example of a factory policy.
+
 [source,json]
 ----
 factory.policies : {
-  "referer"   : STRING,               // Works only for clients from referer
-  "since"     : EPOCHTIME,            // Factory works only after this date
-  "until"     : EPOCHTIME,            // Factory works only before this date
-  "create"    : [perClick | perUser]  // Create one workpace per click, user or account
+  "referer"   : STRING,                 <1>
+  "since"     : EPOCHTIME,              <2>
+  "until"     : EPOCHTIME,              <3>
+  "create"    : [perClick | perUser]    <4>
 }
 ----
+<1> Works only for clients from referer.
+<2> Factory works only after this date.
+<3> Factory works only before this date.
+<4> Create one workpace per click, user, or account.
 
 [id="factories_json_reference-limitations"]
 == Limitations
 
-You can use `since : EPOCHTIME`, `until : EPOCHTIME` and `referer` as a way to prevent the Factory from executing under certain conditions. `since` and `until` represent a valid time window that will allow the Factory to activate. The `referer` will check the hostname of the acceptor and only allow the Factory to execute if there is a match.
+You can use `since : EPOCHTIME`, `until : EPOCHTIME`, and `referer` as a way to prevent the factory from executing under certain conditions. `since` and `until` represent a valid time window that will allow the factory to activate. The `referer` will check the hostname of the acceptor and only allow the factory to execute if there is a match.
```

Comment:
```suggestion
You can use `since : EPOCHTIME`, `until : EPOCHTIME`, and `referer` as a way to prevent the factory from executing under certain conditions. `since` and `until` represent a valid time window that allows the factory to activate. The `referrer` checks the hostname of the acceptor and only allows the factory to execute if there is a match.
```